front/app/containers/Admin/messaging/CustomEmails/CampaignForm/index.tsx

```diff
@@ -125,27 +123,22 @@ const CampaignForm = ({
     return [
       {
         value: 'author',
-        label: !isNilOrError(user) ? getFullName(user.data) : '',
+        label: getFullName(user.data),
       },
       {
         value: 'organization',
-        label: !isNilOrError(appConfig)
-          ? localize(appConfig.data.attributes.settings.core.organization_name)
-          : '',
+        label: localize(
+          appConfig.data.attributes.settings.core.organization_name
+        ),
       },
     ];
   };
 
   const groupsOptions = (groups: IGroupData[]) => {
-    const groupList =
-      !isNilOrError(groups) && !isNilOrError(groups)
-        ? groups.map((group) => ({
-            label: localize(group.attributes.title_multiloc),
-            value: group.id,
-          }))
-        : [];
-
-    return groupList;
+    return groups.map((group) => ({
```

Comment:
If there are no groups, will this .map function still return an empty array? :thinking: Or would it return undefined or null? Wondering if that could impact where groupsOptions is used.

Reply:
Yes. See this line where the function is called:

`options={groupsOptions(groups?.data || [])}`. Map on an empty array returns an empty array.

Code could be clearer, but not touching that for now. Out of scope. :)

Reply:
I actually hadn't checked where it was called, was just going by the types of groupsOptions:

`const groupsOptions = (groups: IGroupData[]) => {`

It says groups is always an array. Given there were no other linter errors, I could build on that type definition without having to check where it was used.

Reply:
Ahh ok true! :) Thanks for pointing that out :+1: